File: ip-messaging/rest/credentials/retrieve-credentials/retrieve-credentials.5.x.rb
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACCOUNT_SID'
auth_token = 'AUTH_TOKEN'
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Retrieve the credential
credential = @client.chat.v2.credentials('CR_SID').fetch
puts credential

File: ip-messaging/rest/credentials/retrieve-credentials/retrieve-credentials.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest.ip_messaging import TwilioIpMessagingClient

# Your Account Sid and Auth Token from twilio.com/user/account
account = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
token = "AUTH_TOKEN"
client = TwilioIpMessagingClient(account, token)

credential = client.credentials.get("CREDENTIAL_SID")

print(credential)
